tests/state_tests.rs

```diff
@@ -0,0 +1,48 @@
+use auction_io::*;
+// use codec::Encode;
+use dutch_auction::{State, StateReply};
+use gtest::System;
+
+mod routines;
+use routines::*;
+
+// #[test]
+// fn is_not_active_after_time_is_over() {
+//     let sys = System::new();
+//
+//     let auction = init(&sys);
+//     sys.spend_blocks(DURATION);
+//
+//     if let StateReply::IsActive(is_active) = auction.meta_state(State::IsActive()).unwrap() {
+//         assert!(!is_active);
+//     } else {
+//         panic!("Can't get state");
+//     }
+// }
```

Comment:
Dead code?
```suggestion
```

Reply:
No, it's not dead, but I can't test it right now because of gtest bug with time)

Reply:
What bug?